en/concept/custom_mixer_payload.md

```diff
@@ -0,0 +1,77 @@
+# Custom Payload Mixers
```

Comment:
So I've tried not to explain too much about mixers here. That should be covered elsewhere ... properly. 
The model is to show how a simple mixer and payloads work in practise, starting from the pas.aux.mix example.